src/Microcosm.js

```diff
@@ -107,8 +107,8 @@ Microcosm.prototype = {
   /**
    * Partially applies `push`.
    */
-  prepare(action, ...params) {
-    return this.push.bind(this, action, ...params)
+  prepare(action, params) {
+    return (more, callback) => this.push(action, [].concat(params, more), callback)
   },
```

Comment:
`prepare` had to change because we no longer have an infinite argument length for `push`. The new signature is:

```javascript
let summer = app.prepare(sum, [ 1, 2 ])

summer(1, function(error, body) {
  assert.equal(body, 4)
})

summer([ 1, 1 ], function(error, body) {
  assert.equal(body, 5)
})